packages/ckeditor5-engine/src/controller/datacontroller.js

```diff
@@ -350,8 +350,13 @@ export default class DataController {
 	 * @fires set
 	 * @param {String|Object.<String,String>} data Input data as a string or an object containing `rootName` - `data`
 	 * pairs to set data on multiple roots at once.
+	 * @param {Object} [options]
```

Comment:
```suggestion
	 * @param {Object} [options={}]
```

As it defines the default value.